template.tpl

```diff
@@ -363,6 +368,54 @@ if (queryPermission('inject_script', url)) {
   log('🧪 Bambuser Debugger 🧪\n', '🚫 Script load failed due to permissions mismatch. You may need to whitelist your custom script URL ('+ url +') in the template setting > permissions > Allowed URL match patterns.');
   data.gtmOnFailure();
 }
+} else if (data.feature === 'oneToOneIntegration') {
+  const conf = {
+    orgId: data.bambuserOrgId,
+    triggers: ['manual', 'connect-link', 'smart'],
+    smartVariantOverride: data.overlayWidgetVariant,
+    popupTimeoutSeconds: makeInteger(data.overlayWidgetPopupTimeuot) || 2,
+    datalayerTracking: data.datalayerTracking,
+    ecommerceTracking: data.ecommerceTracking,
+  };
+  
+  if (!!data.queueId) {
+    conf.queue = data.queueId;
+  }
+  if (!!data.locale) {
+    conf.locale = data.locale;
+  }
+
+  const launch = function (debugMode) {
+    if (queryPermission('access_globals', 'execute', 'launchBambuserOneToOne')) {
+      const oneToOneEmbed = callInWindow('launchBambuserOneToOne', conf, debugMode);
+      log('successfully created one-to-one instance');
+      data.gtmOnSuccess();
+    } else {
+      data.gtmOnFailure();
+    }
+  };
+
+  const url = 'https://one-to-one.bambuser.com/embed.js';
```

Comment:
Maybe we should right away just add a field in the advanced section of the GTM template to choose between EU / US and then given that select url to be `https://one-to-one.bambuser.com/eu/embed.js` or `https://one-to-one.bambuser.com/embed.js`